compensation.adoc

```diff
@@ -66,8 +67,14 @@ See the current BSQ price, open trades, trade history and other information on t
 
 See https://github.com/bisq-network/compensation/issues/277[this link^] for an example compensation request.
 
+==== Wait for team lead review ====
+
+Per the  https://bisq.wiki/Compensation[compensation request review process], your compensation request will be reviewed by the team lead(s) responsible for the work you delievered. You may be asked for further information or to revise your compensation request. For this reason, please do not submit your compensation request to the DAO in the next step until this review process is complete!
```

Comment:
```suggestion
Per the  https://bisq.wiki/Compensation[compensation request review process], your compensation request will be reviewed by the team lead(s) responsible for the work you delivered. You may be asked for further information or to revise your compensation request. For this reason, please do not submit your compensation request to the DAO in the next step until this review process is complete!
```

Reply:
Might it be better to bold or somehow accentuate the part about holding off on submitting a DAO request? Maybe just putting that last sentence on a new line would help. To ensure people see it.

Reply:
> Might it be better to bold or somehow accentuate the part about holding off on submitting a DAO request? Maybe just putting that last sentence on a new line would help. To ensure people see it.

See d134b8c in which I've made the relevant part of the sentence bold. Thanks.